nav2_bt_navigator/include/nav2_bt_navigator/navigate_to_pose_behavior_tree.hpp

```diff
@@ -32,6 +33,7 @@ class NavigateToPoseBehaviorTree
   NavigateToPoseBehaviorTree() = delete;
 
   nav2_tasks::TaskStatus run(
+    BT::Blackboard::Ptr & blackboard,
```

Comment:
A reference to a pointer? Is that correct? Just asking because it seems a bit unusual.